modules/videopress/class.videopress-gutenberg.php

```diff
@@ -164,6 +165,28 @@ public function render_video_block_with_videopress( $attributes, $content ) {
 			1
 		);
 	}
+
+	/**
+	 * Replaces the video uploaded in the block editor.
+	 *
+	 * Enqueues a script that registers an API fetch middleware replacing the video uploads in Gutenberg so they are
+	 * uploaded against the WP.com API media endpoint and thus transcoded by VideoPress.
+	 */
+	public function override_video_upload() {
+		if (
+			method_exists( 'Jetpack', 'is_active' ) && Jetpack::is_active()
```

Comment:
Is this line necessary? Wouldn't the `is_module_active` check be enough?

Reply:
You're right! I simplified the checks in 9d75857